<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="0" ignored="0" total="2" passed="2">
  <reporter-output>
  </reporter-output>
  <suite name="Suite" duration-ms="8298" started-at="2023-04-23T07:48:11Z" finished-at="2023-04-23T07:48:19Z">
    <groups>
    </groups>
    <test name="Test" duration-ms="8298" started-at="2023-04-23T07:48:11Z" finished-at="2023-04-23T07:48:19Z">
      <class name="testcases.LoginTest">
        <test-method status="PASS" signature="ReportSetup()[pri:0, instance:testcases.LoginTest@709ba3fb]" name="ReportSetup" is-config="true" duration-ms="555" started-at="2023-04-23T07:48:11Z" finished-at="2023-04-23T07:48:11Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- ReportSetup -->
        <test-method status="PASS" signature="SetUp(java.lang.reflect.Method)[pri:0, instance:testcases.LoginTest@709ba3fb]" name="SetUp" is-config="true" duration-ms="3097" started-at="2023-04-23T07:48:11Z" finished-at="2023-04-23T07:48:14Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[public void testcases.LoginTest.invalidLogin() throws com.codoid.products.exception.FilloException]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- SetUp -->
        <test-method status="PASS" signature="invalidLogin()[pri:0, instance:testcases.LoginTest@709ba3fb]" name="invalidLogin" duration-ms="522" started-at="2023-04-23T07:48:14Z" finished-at="2023-04-23T07:48:15Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- invalidLogin -->
        <test-method status="PASS" signature="TearDown()[pri:0, instance:testcases.LoginTest@709ba3fb]" name="TearDown" is-config="true" duration-ms="648" started-at="2023-04-23T07:48:15Z" finished-at="2023-04-23T07:48:16Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- TearDown -->
        <test-method status="PASS" signature="SetUp(java.lang.reflect.Method)[pri:0, instance:testcases.LoginTest@709ba3fb]" name="SetUp" is-config="true" duration-ms="2371" started-at="2023-04-23T07:48:16Z" finished-at="2023-04-23T07:48:18Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[public void testcases.LoginTest.validLogin() throws com.codoid.products.exception.FilloException,java.lang.InterruptedException]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- SetUp -->
        <test-method status="PASS" signature="validLogin()[pri:0, instance:testcases.LoginTest@709ba3fb]" name="validLogin" duration-ms="435" started-at="2023-04-23T07:48:18Z" finished-at="2023-04-23T07:48:18Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- validLogin -->
        <test-method status="PASS" signature="TearDown()[pri:0, instance:testcases.LoginTest@709ba3fb]" name="TearDown" is-config="true" duration-ms="647" started-at="2023-04-23T07:48:18Z" finished-at="2023-04-23T07:48:19Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- TearDown -->
      </class> <!-- testcases.LoginTest -->
    </test> <!-- Test -->
  </suite> <!-- Suite -->
</testng-results>
